src/components/sbb-radio-button/sbb-radio-button.stories.js

```diff
@@ -0,0 +1,145 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const name = {
+  control: {
+    type: 'text',
+  },
+};
+
+const value = {
+  control: {
+    type: 'text',
+  },
+};
+
+const checked = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const disabled = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const size = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['m', 's'],
+};
+
+const accessibilityLabel = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Accessibility',
+  },
+};
+
+const accessibilityDescribedby = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Accessibility',
+  },
+};
+
+const accessibilityLabelledby = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'Accessibility',
+  },
+};
+
+const defaultArgTypes = {
+  name,
+  value,
+  checked,
+  disabled,
+  size: size,
+  'accessibility-label': accessibilityLabel,
+  'accessibility-describedby': accessibilityDescribedby,
+  'accessibility-labelledby': accessibilityLabelledby,
+};
+
+const defaultArgs = {
+  name: 'radio-group-name',
+  value: 'First value',
+  checked: false,
+  disabled: false,
+  size: size.options[0],
+  'accessibility-label': undefined,
+  'accessibility-describedby': undefined,
+  'accessibility-labelledby': undefined,
+};
+
+const DefaultTemplate = (args) => <sbb-radio-button {...args}>Value</sbb-radio-button>;
+
+const MultilineLabelTemplate = (args) => (
+  <sbb-radio-button {...args}>
+    I have read and agree to all terms, conditions and notices contained or referenced herein (the
+    “Terms of Service“) and the Privacy Policy Statement (“Privacy Policy“).
```

Comment:
Not sure if this text is more a case for a checkbox. But yes, it's only an example ;)

Reply:
I know, I will try to find another placeholder 😄